internal/controller/api/management_test.go

```diff
@@ -487,6 +493,37 @@ var _ = Describe("Management", func() {
 
 	})
 
+	Describe("Connecting to the reconnect endpoint", func() {
+		Context("With a negative delay", func() {
+			It("Should fail to reconnect", func() {
+
+				postBody := createConnectionReconnectPostBody(CONNECTED_ACCOUNT_NUMBER, CONNECTED_NODE_ID, "-1", "requesting to reconnect")
+
+				req, err := http.NewRequest("POST", CONNECTION_RECONNECT_ENDPOINT, postBody)
+				Expect(err).NotTo(HaveOccurred())
+
+				req.Header.Add(IDENTITY_HEADER_NAME, validIdentityHeader)
+
+				rr := httptest.NewRecorder()
+
+				ms.router.ServeHTTP(rr, req)
+
+				Expect(rr.Code).To(Equal(http.StatusBadRequest))
+
+				var actualResponse errorResponse
+				var expectedResponse = errorResponse{
+					Title:  "Delay field cannot be negative",
```

Comment:
Maybe we should make a constant for these error strings?  Then we can use the constants within the tests and within the main code.  It would remove a bit of duplication.

Reply:
Yeah, that's a good idea!

I'm not sure if those constants should be defined at the top of the main code or in a different package? The config package didn't seem right for them. What do you think? @dehort